py/setup.py

```diff
@@ -72,12 +72,24 @@ def delete_scratchpad():
     # Step 1: build the rust library
     build = spec.add_external_build(cmd=cmd, path=rust_path)
 
+    def find_dylib():
+        try:
+            return build.find_dylib("relay_cabi", in_path="target/%s" % target)
+        except LookupError as e:
```

Comment:
We can probably check for the environment variable first rather than catching the error. That would be less verbose, and also show the intent more clearly.